### ISSUE TITLE
Blog post images

### ISSUE THREAD

**Issue by rossabaker** [typelevel/typelevel.github.com#348]
Most of the blog posts have the same placeholder image.  Not an ideal look.

**Comment by jducoeur** [comment]
True, but not a rare one either (Rally's Engineering blog was in the same state for a long time).  IMO, this isn't a blocker, although it would be nice to go back and improve it later.

**Comment by valencik** [comment]
Addressed in https://github.com/typelevel/typelevel.github.com/pull/413
